core/unit_test/TestMathematicalFunctions3.hpp

```diff
@@ -0,0 +1,21 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+
+#define KOKKOS_MATHEMATICAL_FUNCTIONS_SKIP_1
+#define KOKKOS_MATHEMATICAL_FUNCTIONS_SKIP_2
+#include "TestMathematicalFunctions.hpp"
```

Comment:
It would be preferable to use affirmative definitions for each subset, so that further extensions don't require successively deeper nests of conditions.

Reply:
No.  It was implemented this way at my request.  The rational is that there is less chance of accidentally disabling tests with a typo for instance.